make-it/EmMakefile

```diff
@@ -113,11 +114,22 @@ EM_EXPORTS = -s EXPORTED_FUNCTIONS="[\
     '_Occurrence_Set',\
     '_Data_GetStringBegin',\
     '_Data_GetStringLength',\
-    '_Data_WriteString',\
     '_Data_ReadBoolean',\
+    '_Data_ReadUInt8',\
+    '_Data_ReadUInt16',\
+    '_Data_ReadUInt32',\
```

Comment:
If we are writing the "Data_ReadUInt*" versions, we should write "Data_ReadInt*" versions too (especially as we have the Data_Write* versions for both signed and Unsigned).